### ISSUE TITLE
Cannot resolve org.geoscript:geoscript-groovy:jar:1.6.0 from any repository

### ISSUE THREAD

**Issue by fgdrf** [geoscript/geoscript-groovy#29]
Issue #25 is marked as resolved althought the old version 1.3.1 isn't provided by boundles repository anymore. Now uDig build is failing because 1.6.0 cannot be resolved either anymore.

This raises the question : What is the offical maven release repository for org.geoscript?

From [pom.xml](https://github.com/geoscript/geoscript-groovy/blob/master/pom.xml#L481) its 
`https://boundless.artifactoryonline.com/boundless/release/`
but down below org/geoscript I cannot find any resources.

**Comment by blackrez** [comment]
I found the last version https://repo.boundlessgeo.com/main/org/geoscript/geoscript-groovy/ but I didn't the last stable version.

**Comment by jericks** [comment]
Jody is helping me look into this.  We think it's due to the Boundless Maven repo switch over.  The releases are still on the old server but if we can't get them back, I will republish them.

**Comment by fgdrf** [comment]
Do you mean boundless-cache? I found [this one](https://repo.boundlessgeo.com/webapp/#/artifacts/browse/tree/General/boundlessgeo-cache/org/geoscript/geoscript-groovy/1.6.0/) and couldn't find a public repository URL for it.

**Comment by jericks** [comment]
Frank, with some great help from Jody and the Boundless team, I believe we have this issue solved.

https://repo.boundlessgeo.com/main/org/geoscript/geoscript-groovy/

Can you confirm?

Thanks,
Jared

**Comment by fgdrf** [comment]
Thanks @jericks!
I've tested the maven build for uDig. It resolved GeoScript 1.6.0 sucessfully 
```
Downloaded: https://repo.boundlessgeo.com/main/org/geoscript/geoscript-groovy/1.6.0/geoscript-groovy-1.6.0.jar (2136 KB at 460.3 KB/sec)
```
Thank you very much!
